pyodide_build/pywasmcross.py

```diff
@@ -351,7 +351,11 @@ def calculate_object_exports_readobj(objects: list[str]) -> list[str] | None:
         "-st",
     ] + objects
     completedprocess = subprocess.run(
-        args, encoding="utf8", capture_output=True, env={"PATH": os.environ["PATH"]}
+        args,
+        encoding="utf8",
+        capture_output=True,
+        env={"PATH": os.environ["PATH"]},
+        check=True,
```

Comment:
We check the return code ourselves
```suggestion
        check=False,
```